README.md

```diff
@@ -23,10 +23,21 @@ specification to describe this ability as well, and move the document to the sta
   * todo
 
 ## Goals
-  * todo
+* Enable a web app to register tasks to be run periodically, with a retry logic baked in, which has an exponential backoff.
+* Provide the web app a way to query the list of previously registered tasks.
+* Enable the web app to unregister previously registered tasks.
+* Supported platforms: Android, Windows, Linux, ChromeOS, Mac. 
 
 ## Non-goals
-  * todo
+* Triggering events at a specific time is an explicit non-goal. The alarms API enables that.
+* Notifying the user for each periodic Sync event. The user can be presented with updated content
+  when they next visit the website.
+* Unsupported platforms: iOS, WebView. This API requires a way to wake up the service worker, but
+  in a WebView, the browser cannot start itself, and instead it only starts when the embedding app embeds a WebView.
+  In addition, it's hard to identify the correct WebView to deliver the periodic sync event to.
```

Comment:
As above - that's specific to our implementation. Android and iOS can absolutely adopt the standard and make the necessary changes to their platforms to enable this.

Reply:
removed.